<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Анализатор дискового пространства</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="controlLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Путь:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="pathEdit">
        <property name="placeholderText">
         <string>Введите путь для сканирования</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="browseBtn">
        <property name="text">
         <string>Обзор...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="scanBtn">
        <property name="text">
         <string>Сканировать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopBtn">
        <property name="text">
         <string>Остановить</string>
        </property>
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="progressLayout">
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
        <property name="visible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>Готов к сканированию</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <!-- Убрали вкладку с Treemap пока -->
      <widget class="QWidget" name="tabChart">
       <attribute name="title">
        <string>Диаграмма</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QtCharts::QChartView" name="chartView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabFiles">
       <attribute name="title">
        <string>Большие файлы</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTableWidget" name="filesTable">
		   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
  </property>
  <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
  </property>
          <property name="columnCount">
           <number>4</number>
          </property>
          <property name="rowCount">
           <number>0</number>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Имя файла</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Размер</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Путь</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Дата изменения</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts/QChartView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>